Inference Code

hamzakhalil798 opened the pull request:
Hey! I've added the Inference part to the code.
Changes :
Added a new class in dataloader.py for visualization. It loads two images instead of All images in the dataset.
Added visualize_DRAEM.py, it's basically the same as test_DRAEM.py with minor changes to display Original Images and Resulting Anomaly Heatmaps.

hamzakhalil798 committed:
Added Visualization Code

hamzakhalil798 committed:
Added snapshots of result

hamzakhalil798 committed:
Update README.md

hamzakhalil798 committed:
Updated README.md

added Inference Discription

hamzakhalil798 committed:
Update README.md

adbensi commented:
[hamzakhalil798] Please, wouldn't it be more common to just upload an image for inference rather than uploading files from the training folder itself?

MAwaisr committed:
Update README.md

chmod +x command is added for clarity to avoid permission denial while downloading datasets and pretrained models

hamzakhalil798 committed:
Merge pull request #2 from MAwaisr/patch-1

Update README.md
